puppet/modules/chassis/manifests/wp.pp

```diff
@@ -68,10 +68,8 @@
 		ensure => 'enabled',
 	}
 
-	if ( $theme ) {
-		wp::theme { $theme:
-			location => $location,
-			ensure => 'enabled',
-		}
+	wp::theme { $themes:
+		location => $location,
+		ensure => 'enabled',
```

Comment:
Reckon you could add another space to line up the `=>` again. Sorry to be so anal about it 😄